packages/dmn-js-decision-table/assets/css/dmn-js-decision-table.css

```diff
@@ -317,6 +317,16 @@
   text-overflow: ellipsis;
 }
 
+.dmn-decision-table-container thead .input-expression:empty:before {
+  content: "Input";

```

Comment:
does it have to be double quotes? We use single quotes in other places

Reply:
We use double quotes in HTML/JSX and single quotes in all other places.

Reply:
I'll change that for sake of consistency.